dpypelines/s3_tar_received.py

```diff
@@ -46,56 +46,25 @@ def start(s3_object_name: str):
             )
         ) from err
 
-    # Check for the existence of a configuration file
+    # get_dataset_id() currently empty - returns "not-specified"
+    # To be updated once we know where the dataset_id can be extracted from (not necessarily s3_object_name as suggested by argument name)
+    dataset_id = get_dataset_id(s3_object_name)
```

Comment:
needs a catch, if this fails for whatever reason we'll need to be notified.

Reply:
I know it cant fail _now_ btw, but lets be consistent, all processes should have a catch etc.